### PR TITLE
Removed the border-radius on align left

### PR DESCRIPTION
**Pull request by Spoorthy1423**
#741 Removed unused sections and

Removed border-radius when the thumbnail display is align left using thumbnailAlign prop and made thumbnail prop large.

#### Issue addressed

Addresses #*PR# HERE*

### Before/after screenshots

After changes
Screenshot 2024-08-24 at 6 43 33 PM

## Changelog

- [#PR no]
  - **Description:** Summary of change(s)
  - **Products impact:** Choose from - none (for internal updates) / bugfix / new API / updated API / removed API. If it's 'none', use "-" for all items below to indicate they are not relevant.
  - **Addresses:** Link(s) to GH issue(s) addressed. Include KDS links as well as links to related issues in a consumer product repository too.
  - **Components:** Affected public KDS component. Do not include internal sub-components or documentation components.
  - **Breaking:** Will this change break something in a consumer? Choose from: yes / no
  - **Impacts a11y:** Does this change improve a11y or adds new features that can be used to improve it? Choose from: yes / no
  - **Guidance:** Why and how to introduce this update to a consumer? Required for breaking changes, appreciated for changes with a11y impact, and welcomed for non-breaking changes when relevant.

[#PR no]: PR link

## Steps to test

1. Step 1
2. Step 2
3. ...

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [ ] The change is described in the changelog section above

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## After review

- [ ] The changelog item has been pasted to the `CHANGELOG.md`

### CODE REVIEW

**Comment by Spoorthy1423** [comment]
Hey!! @AllanOXDi can u review my pr please, i would like to work on this repo further!!

File: package-lock.json
Comment (AllanOXDi) [inline package-lock.json:1]: Hi @Spoorthy1423   can we not commit this file?

File: lib/KCard/index.vue [hunk 1]
```diff
@@ -320,7 +320,7 @@
             },
             thumbnailStyles: {
               ...thumbnailCommonStyles,
-              borderRadius: '8px 0 0 8px',
+              borderRadius: '0 0 0 0',
```
Comment (AllanOXDi) [inline lib/KCard/index.vue:323]: I think let's just remove this  borderRadius  property  from here . it  will still achieve  same result you are trying to achieve here. or you could simplify it  to `borderRadius: 0,`

**Review by AllanOXDi — CHANGES_REQUESTED** [review]

**Review by AllanOXDi — CHANGES_REQUESTED** [review]
Could you as well  add your PR details to  the [CHANGELOG.md](https://github.com/learningequality/kolibri-design-system/pull/736/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed)to make make sure all the check pass . Here is an example https://github.com/learningequality/kolibri-design-system/pull/736/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR6

File: package-lock.json
Comment (Spoorthy1423) [reply package-lock.json:1]: Sure

File: lib/KCard/index.vue (hunk 1 above)
Comment (Spoorthy1423) [reply lib/KCard/index.vue:323]: Do u want me to raise another pr with required changes??

**Pull request closed without merging**